ui/src/app/components/rstudio-configuration-panel.spec.tsx

```diff
@@ -22,39 +22,38 @@ import {
 
 import { defaultRStudioConfig } from './apps-panel/utils';
 import {
-  BaseRStudioConfigurationPanel,
+  RStudioConfigurationPanel,
   RStudioConfigurationPanelProps,
 } from './rstudio-configuration-panel';
 
-describe('RStudioConfigurationPanel', () => {
-  const onClose = jest.fn();
-  const freeTierBillingAccountId = 'freetier';
-
-  const DEFAULT_PROPS: RStudioConfigurationPanelProps = {
-    onClose,
-    creatorFreeCreditsRemaining: null,
-    workspace: {
-      ...workspaceStubs[0],
-      accessLevel: WorkspaceAccessLevel.WRITER,
-      billingAccountName: 'billingAccounts/' + freeTierBillingAccountId,
-      cdrVersionId: CdrVersionsStubVariables.DEFAULT_WORKSPACE_CDR_VERSION_ID,
-    },
-    profileState: {
-      profile: ProfileStubVariables.PROFILE_STUB,
-      load: jest.fn(),
-      reload: jest.fn(),
-      updateCache: jest.fn(),
-    },
-    gkeAppsInWorkspace: [],
-  };
+const onClose = jest.fn();
+const freeTierBillingAccountId = 'freetier';
+export const DEFAULT_PROPS: RStudioConfigurationPanelProps = {
+  onClose,
+  creatorFreeCreditsRemaining: null,
+  workspace: {
+    ...workspaceStubs[0],
+    accessLevel: WorkspaceAccessLevel.WRITER,
+    billingAccountName: 'billingAccounts/' + freeTierBillingAccountId,
+    cdrVersionId: CdrVersionsStubVariables.DEFAULT_WORKSPACE_CDR_VERSION_ID,
+  },
+  profileState: {
+    profile: ProfileStubVariables.PROFILE_STUB,
+    load: jest.fn(),
+    reload: jest.fn(),
+    updateCache: jest.fn(),
+  },
+  gkeAppsInWorkspace: [],
+};
```

Comment:
This is the same code as before but moved out of the `describe` block and `export`ed for use in `gke-app-configuration-panel.spec.tsx`.